### PR TITLE
fix some copy-pasting error

### PR DESCRIPTION
**Pull request by tycho01**
see title / commit

### CODE REVIEW

File: src/compiler/types.ts [hunk 1]
```diff
@@ -3126,7 +3126,7 @@ namespace ts {
         /* @internal */
         ContainsObjectLiteral   = 1 << 22,  // Type is or contains object literal type
         /* @internal */
-        ContainsAnyFunctionType = 1 << 23,  // Type is or contains object literal type
+        ContainsAnyFunctionType = 1 << 23,  // Type is or contains a function type
```
Comment (ahejlsberg) [inline src/compiler/types.ts]: Actually, this should be "Type is or contains the anyFunctionType".

Comment (tycho01) [inline src/compiler/types.ts]: Thanks, fixed it now.

**Comment by tycho01** [comment]
Not sure how changing a comment made the build fail.

**Comment by RyanCavanaugh** [comment]
Restarting Travis on the assumption someone else broke the build and has fixed it in the interim

**Pull request merged**